### PR TITLE
Playbooks to Setup Quay

### PR DESCRIPTION
**Pull request by nasx**

### CODE REVIEW

**Comment by chadmf** [comment]
We should probably specify that you must be on openshift version > 4.7 since the gitops operator does not work on 4.6 unless they GA the operator on older versions this will break for anyone trying on an older version.

**Comment by nasx** [comment]
> We should probably specify that you must be on openshift version > 4.7 since the gitops operator does not work on 4.6 unless they GA the operator on older versions this will break for anyone trying on an older version.

Agreed. I'll add a card to Trello.

File: ansible/playbooks/quay-setup.yaml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        api_version: quay.redhat.com/v1
+        ca_cert: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        host: https://kubernetes.default.svc
+        kind: QuayRegistry
+        name: "{{ quay_registry_name }}"
+        namespace: "{{ quay_namespace }}"
+        validate_certs: yes
+      delay: 10
+      register: quay_registry_response
+      retries: 60
```
[the remark this answers is not in the document]
Comment (nasx) [reply ansible/playbooks/quay-setup.yaml:98]: All of the Jobs we have run `PostSync` w/ `activeDeadlineSeconds` set to 600. I was just following that pattern. In my testing Quay typically deploys in < 5 minutes. What were you thinking of increasing it to? Not opposed, just explaining my rationale.

[the remark this answers is not in the document]
Comment (sabre1041) [reply ansible/playbooks/quay-setup.yaml:98]: im thinking slow networks.

[the remark this answers is not in the document]
Comment (sabre1041) [reply ansible/playbooks/quay-setup.yaml:98]: maybe double to 120?

[the remark this answers is not in the document]
Comment (nasx) [reply ansible/playbooks/quay-setup.yaml]: Done.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**